src/dotnet/Chat.UI.Blazor/Components/ChatView/Items/ChatMessage.razor

```diff
@@ -239,6 +243,18 @@
         return hasAnyMentions && MarkupValidator.ContainsMention(new MentionId(authorId, AssumeValid.Option)).IsValid(markup);
     }
 
+    private (List<TextEntryAttachment>, List<TextEntryAttachment>) GetOrderedAttachmentList(ChatEntry entry) {
+        var imageList = new List<TextEntryAttachment>();
+        var fileList = new List<TextEntryAttachment>();
+        foreach (var attachment in entry.Attachments) {
```

Comment:
attachment is very small, so I suggest using 2 linq queries instead of this code